ui/js/controllers/main.js

```diff
@@ -107,6 +107,25 @@ treeherderApp.controller('MainCtrl', [
             }
         };
 
+        /* If the user is in an editable element or pressing shift
+         * then disable keyboard events, unless otherwise enabled
+         * in inputs by the 'mousetrap' class in markup */
+        Mousetrap.stopCallback = function(ev, element) {
+            // if the element has the class "mousetrap" then no need to stop
+            if (element.classList.contains('mousetrap')) {
+                return false;
+            }
+            if ((element.tagName === 'INPUT' &&
+                 element.type !== "radio" && element.type !== "checkbox") ||
+                element.tagName === 'SELECT' ||
+                element.tagName === 'TEXTAREA' ||
+                element.isContentEditable || ev.keyCode === 16) {
+                return true;
+            }
+            return false;
+        };
```

Comment:
So I'm not sure about this, as you're overriding global behaviour inside a local controller. That doesn't sound right. I would say either this should be done at a higher level in treeherder, or we should use a local override here -- it sounds like we can do that by adding "mousetrap" to the elements we don't want to interfere with keyboard navigation.

https://github.com/ccampbell/mousetrap/issues/165#issuecomment-28713244